tasks/deploy/github.py

```diff
@@ -0,0 +1,172 @@
+import errno
+import json
+import os
+import re
+
+from invoke.exceptions import Exit
+
+from .githubapp import GithubApp, GithubAppException
+
+errno_regex = re.compile(r".*\[Errno (\d+)\] (.*)")
+
+__all__ = ["Github", "GithubException"]
+
+
+class GithubException(Exception):
+    pass
+
+
+class Github(object):
+    BASE_URL = "https://api.github.com"
+
+    def __init__(self, api_token=None):
+        self.api_token = api_token if api_token else self._api_token()
+
+    def test_repo_found(self, repo):
+        """
+        Checks if a repo can be found. This is useful for testing access permissions to repos.
+        """
+        result = self.repo(repo)
+
+        # name is arbitrary, just need to check if something is in the result
+        if "name" in result:
+            return
+
+        print("Cannot find Github repo {}".format(repo))
+        print("If you cannot see it in the Github UI, you likely need permission.")
+        raise Exit(code=1)
```

Comment:
Should this be a GithubException? Since this could be used outside of invoke.
```suggestion
        raise GithubException("Cannot find Github repo {}".format(repo))
```

Reply:
True. I found out that this method is not used at all anymore, so I'll remove it.